apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -93,9 +96,9 @@ suite('thread_list_ui', function() {
         '<li id="thread-' + i + '" class="threadlist-item" ' +
             'data-time="' + someDate + '" ' +
             'data-thread-id="' + i + '">' +
-          '<label>' +
-            '<input type="checkbox" data-mode="threads" value="' + i + '"/>' +
-          '</label>' +
+          '<gaia-checkbox data-mode="threads" value="' + i + '">' +
```

Comment:
Currently we're implementing all components as `<gaia-[componentName]>`, like gaia-header. I think this is more in line with the other components that we currently have implemented, but I'm not against changing (if we do so everywhere).